test/crdsvalidation/dataplane_test.go

```diff
@@ -60,6 +60,73 @@ func TestDataPlane(t *testing.T) {
 					},
 				},
 			},
+			{
+				Name: "dbmode off is supported",
```

Comment:
How about verifying that an empty string is allowed as well?

Reply:
👍 7c6ccc6db6670630650db94451113656d8658c2e